packages/components/src/combobox-control/style.scss

```diff
@@ -32,7 +32,7 @@ input.components-combobox-control__input[type="text"] {
 	align-items: flex-start;
 	width: 100%;
 	margin: 0 0 $grid-unit-10 0;
-	padding: $grid-unit-05;
+	padding: 0;
```

Comment:
While the padding left/right is now being set in the `styles.ts` file, it doesn't look like the bottom/left padding is being set anywhere else — and therefore, it's just being changed to `0`. Is that on purpose?

Reply:
Yep. Setting a padding here will necessitate the suggestions list to have a hacky negative margin to break out of it. (I fixed it in `FormTokenField`, and `ComboboxControl` reuses that suggestions list component.)